roles/lib_utils/filter_plugins/oo_filters.py

```diff
@@ -767,4 +767,5 @@ def filters(self):
             "lib_utils_oo_etcd_host_urls": lib_utils_oo_etcd_host_urls,
             "lib_utils_mutate_htpass_provider": lib_utils_mutate_htpass_provider,
             "lib_utils_oo_oreg_image": lib_utils_oo_oreg_image,
+            "oo_flatten": oo_flatten,
```

Comment:
Is this supposed to be part of another patch set?

Reply:
This is the only place where `oo_flatten` is used as filter, otherwise its being used as a function (in `lib_utils_oo_select_keys_from_list`).

I could split this change into a new PR - however tracking dependencies would be a bit more complex

Reply:
Sorry, I didn't notice the occurrence of oo_flatten elsewhere in this patch, it's fine.